README.md

```diff
@@ -86,6 +86,8 @@ task test -- -run 'TestAcc_Project*'
 ## Building The Documentation
 
 ```sh
+export GOBIN=$PWD/bin
```

Comment:
Not a fan of this as it makes this repo the location of all global go binary installs. 

Maybe should be ~/go/bin